tests/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests.csproj

```diff
@@ -9,6 +9,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="StreamJsonRpc" />
```

Comment:
Why do we need `StreamJsonRpc` everywhere?

Reply:
This is what I hinted at with:

> gets things building by adding a few package references in order to break the tie on some assembly version conflicts during build.

Without this, restore completes successfully but the build fails with:

```
Found conflicts between different versions of "StreamJsonRpc" that could not be resolved.
There was a conflict between "StreamJsonRpc, Version=2.16.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" and "StreamJsonRpc, Version=2.18.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a".
    "StreamJsonRpc, Version=2.16.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was chosen because it was primary and "StreamJsonRpc, Version=2.18.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was not.
    References which depend on "StreamJsonRpc, Version=2.16.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" [D:\NuGetPackages\streamjsonrpc\2.16.41\lib\netstandard2.0\StreamJsonRpc.dll].
        D:\NuGetPackages\streamjsonrpc\2.16.41\lib\netstandard2.0\StreamJsonRpc.dll
          Project file item includes which caused reference "D:\NuGetPackages\streamjsonrpc\2.16.41\lib\netstandard2.0\StreamJsonRpc.dll".
            D:\NuGetPackages\streamjsonrpc\2.16.41\lib\netstandard2.0\StreamJsonRpc.dll
    References which depend on or have been unified to "StreamJsonRpc, Version=2.18.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" [].
        D:\NuGetPackages\microsoft.servicehub.framework\4.4.23\lib\net472\Microsoft.ServiceHub.Framework.dll
          Project file item includes which caused reference "D:\NuGetPackages\microsoft.servicehub.framework\4.4.23\lib\net472\Microsoft.ServiceHub.Framework.dll".
            D:\NuGetPackages\microsoft.servicehub.framework\4.4.23\lib\net472\Microsoft.ServiceHub.Framework.dll
            D:\NuGetPackages\microsoft.visualstudio.languageservices\4.11.0-1.24225.10\lib\net472\Microsoft.VisualStudio.LanguageServices.dll
            D:\NuGetPackages\microsoft.visualstudio.projectsystem.query\17.8.53-pre\lib\netstandard2.0\Microsoft.VisualStudio.ProjectSystem.Query.dll
            D:\NuGetPackages\microsoft.visualstudio.rpccontracts\17.10.3-preview\lib\netstandard2.0\Microsoft.VisualStudio.RpcContracts.dll
            D:\NuGetPackages\microsoft.visualstudio.editor\17.0.32-g9d6b2c6f26\lib\net472\Microsoft.VisualStudio.Editor.dll
            D:\NuGetPackages\microsoft.visualstudio.shell.15.0\17.3.32804.24\lib\net472\Microsoft.VisualStudio.Shell.15.0.dll
            D:\NuGetPackages\microsoft.visualstudio.hotreload.components\6.0.0-preview.6.21380.4\lib\net472\Microsoft.VisualStudio.HotReload.Components.dll
            D:\NuGetPackages\microsoft.visualstudio.projectsystem.vs\17.8.53-pre\lib\net472\Microsoft.VisualStudio.ProjectSystem.VS.dll
            D:\NuGetPackages\microsoft.visualstudio.shell.design\17.3.32804.24\lib\net472\Microsoft.VisualStudio.Shell.Design.dll
            D:\NuGetPackages\microsoft.visualstudio.shell.framework\17.3.32804.24\lib\net472\Microsoft.VisualStudio.Shell.Framework.dll
            D:\NuGetPackages\microsoft.visualstudio.utilities\17.7.36307-preview.2\lib\net472\Microsoft.VisualStudio.Utilities.dll
            D:\NuGetPackages\microsoft.visualstudio.imaging\17.3.32804.24\lib\net472\Microsoft.VisualStudio.Imaging.dll
            D:\NuGetPackages\microsoft.visualstudio.projectsystem\17.8.53-pre\lib\net472\Microsoft.VisualStudio.ProjectSystem.dll
            d:\repos\project-system\artifacts\Debug\bin\Dlls\net472\Microsoft.VisualStudio.ProjectSystem.Managed.dll
        D:\NuGetPackages\microsoft.visualstudio.language\17.0.32-g9d6b2c6f26\lib\net472\Microsoft.VisualStudio.Language.dll
          Project file item includes which caused reference "D:\NuGetPackages\microsoft.visualstudio.language\17.0.32-g9d6b2c6f26\lib\net472\Microsoft.VisualStudio.Language.dll".
            D:\NuGetPackages\microsoft.visualstudio.language\17.0.32-g9d6b2c6f26\lib\net472\Microsoft.VisualStudio.Language.dll
            D:\NuGetPackages\microsoft.visualstudio.editor\17.0.32-g9d6b2c6f26\lib\net472\Microsoft.VisualStudio.Editor.dll
            D:\NuGetPackages\microsoft.visualstudio.languageservices\4.11.0-1.24225.10\lib\net472\Microsoft.VisualStudio.LanguageServices.dll
        D:\NuGetPackages\microsoft.visualstudio.languageservices\4.11.0-1.24225.10\lib\net472\Microsoft.VisualStudio.LanguageServices.dll
          Project file item includes which caused reference "D:\NuGetPackages\microsoft.visualstudio.languageservices\4.11.0-1.24225.10\lib\net472\Microsoft.VisualStudio.LanguageServices.dll".
            D:\NuGetPackages\microsoft.visualstudio.languageservices\4.11.0-1.24225.10\lib\net472\Microsoft.VisualStudio.LanguageServices.dll
        D:\NuGetPackages\microsoft.visualstudio.projectsystem.query\17.8.53-pre\lib\netstandard2.0\Microsoft.VisualStudio.ProjectSystem.Query.dll
          Project file item includes which caused reference "D:\NuGetPackages\microsoft.visualstudio.projectsystem.query\17.8.53-pre\lib\netstandard2.0\Microsoft.VisualStudio.ProjectSystem.Query.dll".
            D:\NuGetPackages\microsoft.visualstudio.projectsystem.query\17.8.53-pre\lib\netstandard2.0\Microsoft.VisualStudio.ProjectSystem.Query.dll
        D:\NuGetPackages\microsoft.visualstudio.rpccontracts\17.10.3-preview\lib\netstandard2.0\Microsoft.VisualStudio.RpcContracts.dll
          Project file item includes which caused reference "D:\NuGetPackages\microsoft.visualstudio.rpccontracts\17.10.3-preview\lib\netstandard2.0\Microsoft.VisualStudio.RpcContracts.dll".
            D:\NuGetPackages\microsoft.visualstudio.rpccontracts\17.10.3-preview\lib\netstandard2.0\Microsoft.VisualStudio.RpcContracts.dll
            D:\NuGetPackages\microsoft.visualstudio.editor\17.0.32-g9d6b2c6f26\lib\net472\Microsoft.VisualStudio.Editor.dll
            D:\NuGetPackages\microsoft.visualstudio.languageservices\4.11.0-1.24225.10\lib\net472\Microsoft.VisualStudio.LanguageServices.dll
            D:\NuGetPackages\microsoft.visualstudio.projectsystem.query\17.8.53-pre\lib\netstandard2.0\Microsoft.VisualStudio.ProjectSystem.Query.dll
            D:\NuGetPackages\microsoft.visualstudio.shell.15.0\17.3.32804.24\lib\net472\Microsoft.VisualStudio.Shell.15.0.dll
            D:\NuGetPackages\microsoft.visualstudio.hotreload.components\6.0.0-preview.6.21380.4\lib\net472\Microsoft.VisualStudio.HotReload.Components.dll
            D:\NuGetPackages\microsoft.visualstudio.projectsystem.vs\17.8.53-pre\lib\net472\Microsoft.VisualStudio.ProjectSystem.VS.dll
            D:\NuGetPackages\microsoft.visualstudio.shell.design\17.3.32804.24\lib\net472\Microsoft.VisualStudio.Shell.Design.dll
            D:\NuGetPackages\microsoft.visualstudio.utilities\17.7.36307-preview.2\lib\net472\Microsoft.VisualStudio.Utilities.dll
            D:\NuGetPackages\microsoft.visualstudio.imaging\17.3.32804.24\lib\net472\Microsoft.VisualStudio.Imaging.dll
            D:\NuGetPackages\microsoft.visualstudio.projectsystem\17.8.53-pre\lib\net472\Microsoft.VisualStudio.ProjectSystem.dll
            d:\repos\project-system\artifacts\Debug\bin\Dlls\net472\Microsoft.VisualStudio.ProjectSystem.Managed.dll
            D:\NuGetPackages\microsoft.visualstudio.shell.framework\17.3.32804.24\lib\net472\Microsoft.VisualStudio.Shell.Framework.dll
        D:\NuGetPackages\microsoft.visualstudio.utilities\17.7.36307-preview.2\lib\net472\Microsoft.VisualStudio.Utilities.dll
          Project file item includes which caused reference "D:\NuGetPackages\microsoft.visualstudio.utilities\17.7.36307-preview.2\lib\net472\Microsoft.VisualStudio.Utilities.dll".
            D:\NuGetPackages\microsoft.visualstudio.utilities\17.7.36307-preview.2\lib\net472\Microsoft.VisualStudio.Utilities.dll
            D:\NuGetPackages\microsoft.visualstudio.imaging\17.3.32804.24\lib\net472\Microsoft.VisualStudio.Imaging.dll
            D:\NuGetPackages\microsoft.visualstudio.languageservices\4.11.0-1.24225.10\lib\net472\Microsoft.VisualStudio.LanguageServices.dll
            D:\NuGetPackages\microsoft.visualstudio.projectsystem\17.8.53-pre\lib\net472\Microsoft.VisualStudio.ProjectSystem.dll
            d:\repos\project-system\artifacts\Debug\bin\Dlls\net472\Microsoft.VisualStudio.ProjectSystem.Managed.dll
            D:\NuGetPackages\microsoft.visualstudio.projectsystem.vs\17.8.53-pre\lib\net472\Microsoft.VisualStudio.ProjectSystem.VS.dll
            D:\NuGetPackages\microsoft.visualstudio.shell.15.0\17.3.32804.24\lib\net472\Microsoft.VisualStudio.Shell.15.0.dll
            D:\NuGetPackages\microsoft.visualstudio.hotreload.components\6.0.0-preview.6.21380.4\lib\net472\Microsoft.VisualStudio.HotReload.Components.dll
            D:\NuGetPackages\microsoft.visualstudio.shell.design\17.3.32804.24\lib\net472\Microsoft.VisualStudio.Shell.Design.dll
            D:\NuGetPackages\microsoft.visualstudio.shell.framework\17.3.32804.24\lib\net472\Microsoft.VisualStudio.Shell.Framework.dll
            D:\NuGetPackages\microsoft.visualstudio.editor\17.0.32-g9d6b2c6f26\lib\net472\Microsoft.VisualStudio.Editor.dll
```

That error occurred for every project I added an explicit package reference to `StreamJsonRpc`.